src/logger.ts

```diff
@@ -43,6 +43,7 @@ interface VercelData {
 export class Logger {
   public logEvents: LogEvent[] = [];
   throttledSendLogs = throttle(this.sendLogs, 1000);
+  _children: Logger[] = [];
```

Comment:
Do we need the underscore? `throttledSendLogs` doesn't have it and is also private

Reply:
we don't need it, removed it.